public/index.html

```diff
@@ -2,6 +2,11 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
+    <meta name="description" content="Polyfill bundle"/>
```

Comment:
Maybe something like: select an API you want to pollyfill and include it via our CDN. 

What do you think?

Reply:
Yup, I will add that soon.